### PR TITLE
feat: Show scrollbar when the content is taller that then window

### PR DESCRIPTION
**Pull request by mohandast52**
## Proposed changes

- Added custom scrollbar globally.  
- On clicking the "Add funds" button, it will automatically scroll to the bottom.

https://github.com/user-attachments/assets/e1170d01-39ce-4df4-bada-778492a9d093

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: frontend/components/MainPage/sections/AddFundsSection.tsx [hunk 1]
```diff
@@ -35,13 +35,21 @@ const CustomizedCardSection = styled(CardSection)<{ border?: boolean }>`
 export const AddFundsSection = () => {
   const [isAddFundsVisible, setIsAddFundsVisible] = useState(false);
 
+  const addFunds = useCallback(async () => {
+    setIsAddFundsVisible(true);
+
+    await new Promise((resolve) => setTimeout(resolve, 100));
```
Comment (Tanya-atatakai) [inline frontend/components/MainPage/sections/AddFundsSection.tsx:41]: I think I've seen it already in previous PRs, maybe make sense to implement a `delay` function so it's easier to read?

**Review by Tanya-atatakai — APPROVED** [review]

**Review by iasonrovis — APPROVED** [review]

**Review by oaksprout — COMMENTED** [review]
This is pretty janky, would be good to resolve somehow: image

Can you confirm that the scrollbar doesn't show in the happy state, this in this screenshot:
image

**Review by truemiller — APPROVED** [review]

**Comment by truemiller** [comment]
unsure if it's worth making only below the header scrollable?
i.e. with overflow-y

**Comment by oaksprout** [comment]
> unsure if it's worth making only below the header scrollable? i.e. with overflow-y

Yes I had the same thought.

**Review by oaksprout — APPROVED** [review]

**Pull request merged**